remote_redis_test.go

```diff
@@ -0,0 +1,554 @@
+package skipper_test
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"net"
+	"net/http"
+	stdlibhttptest "net/http/httptest"
+	"net/url"
+	"os"
+	"syscall"
+	"testing"
+	"time"
+
+	"github.com/zalando/skipper"
+	"github.com/zalando/skipper/dataclients/kubernetes/kubernetestest"
+	"github.com/zalando/skipper/filters"
+	flog "github.com/zalando/skipper/filters/accesslog"
+	fscheduler "github.com/zalando/skipper/filters/scheduler"
+	"github.com/zalando/skipper/loadbalancer"
+	"github.com/zalando/skipper/metrics/metricstest"
+	"github.com/zalando/skipper/net/httptest"
+	"github.com/zalando/skipper/net/redistest"
+	"github.com/zalando/skipper/proxy"
+	"github.com/zalando/skipper/routesrv"
+	"github.com/zalando/skipper/routing"
+	"github.com/zalando/skipper/routing/testdataclient"
+	"github.com/zalando/skipper/scheduler"
+	"github.com/zalando/skipper/tracing/tracingtest"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestConcurrentKubernetesClusterStateAccessWithRemoteRedis(t *testing.T) {
```

Comment:
I think you only need this test case and others could be left in skipper package

Reply:
Fixed by https://github.com/zalando/skipper/pull/2419